public/controllers/management/components/management/ruleset/ruleset-editor.js

```diff
@@ -112,27 +117,33 @@ class WzRulesetEditor extends Component {
       try {
         await validateConfigAfterSent();
       } catch (error) {
-        const toast = {
-          title: 'File content is incorrect.',
-          toastMessage: `The content of the file ${name} is incorrect. There were found several error while validating the configuration.`,
-          toastLifeTimeMs: 5000,
+        const options = {
+          context: errorContext,
+          level: UI_LOGGER_LEVELS.ERROR,
+          severity: UI_ERROR_SEVERITIES.BUSINESS,
+          error: {
+            error: error,
+            message:`The content of the file ${name} is incorrect. There were found several error while validating the configuration: ${error.message || error}`,
+            title: `Error file content is incorrect: ${error.message || error}`,
+          },
         };
-
+        getErrorOrchestrator().handleError(options);
         this.setState({ isSaving: false });
         this.goToEdit(name);
 
+        let toastMessage;
+
         if (this.props.state.addingRulesetFile != false) {
           //remove current invalid file if the file is new.
           await this.rulesetHandler.deleteFile(name);
-          toast.toastMessage += '\nThe new file was deleted.';
+          toastMessage = 'The new file was deleted.';
         } else {
           //restore file to previous version
           await this.rulesetHandler.updateFile(name, this.state.initContent, overwrite);
-          toast.toastMessage += '\nThe content file was restored to previous state.';
+          toastMessage = 'The content file was restored to previous state.';
         }
 
-        getToasts().addError({ stack: error, message: toast.toastMessage }, toast);
-
+        this.showToast('success', 'Success', toastMessage, 3000);
```

Comment:
Here you have 2 logics to handle the error. Should be used only our orquestrator. So you can define the message using this logic but remove the other toasts.
For example: 
```
    this.setState({ isSaving: false });
    this.goToEdit(name);

    let errorMessage = `The content of the file ${name} is incorrect. There were found several error while validating the configuration: ${error.message || error}`;
    if (this.props.state.addingRulesetFile != false) {
      //remove current invalid file if the file is new.
      await this.rulesetHandler.deleteFile(name);
      errorMessage += '\nThe new file was deleted.';
    } else {
      //restore file to previous version
      await this.rulesetHandler.updateFile(name, this.state.initContent, overwrite);
      errorMessage += '\nThe content file was restored to previous state.';
    }

    const options = {
      context: errorContext,
      level: UI_LOGGER_LEVELS.ERROR,
      severity: UI_ERROR_SEVERITIES.BUSINESS,
      error: {
        error: error,
        message: errorMessage,
        title: `Error file content is incorrect: ${error.message || error}`,
      },
    };
    getErrorOrchestrator().handleError(options);
```